addons/phone_validation/models/res_partner.py

```diff
@@ -30,3 +36,15 @@ def _phone_format(self, number, country=None, company=None):
             force_format='INTERNATIONAL',
             raise_exception=False
         )
+
+    # FIXME DRY
+    def _phone_get_country_field(self):
+        if 'country_id' in self:
```

Comment:
Actually as we know it is available in base, we could directly return 'country_id'